dataprofiler/profilers/profile_builder.py

```diff
@@ -2240,7 +2240,20 @@ def _update_null_replication_metrics(self, clean_samples: Dict) -> None:
         :param clean_samples: input cleaned dataset
         :type clean_samples: dict
         """
-        data = pd.DataFrame(clean_samples).apply(pd.to_numeric, errors="coerce")
+        data = pd.DataFrame(clean_samples)
+
+        # If the last row is all null, then add rows to the data DataFrame
+        max_null_index = max(
+            [max(i) for i in getattr(self._profile[0], "null_types_index").values()],
```

Comment:
`self._profile[0]` Right now this is getting the first profile every time is this correct? 

Why do we need to add rows to the df?

Reply:
If the last row(s) is all null, then the maximum null index of any profile should be the index of the last row.

Reply:
- in this test case, `len(self._profile)` is `3` and only the zero-th index has values.
```
getattr(self._profile[0], "null_types_index").values()
dict_values([{2, 4}])
getattr(self._profile[1], "null_types_index").values()
dict_values([])
getattr(self._profile[2], "null_types_index").values()
dict_values([])
```

@tonywu315 is this always the case though? are we sure we can always just pop pull the zero-th profile in the `self._profile` list?